content/en/monitors/monitor_types/forecasts.md

```diff
@@ -9,39 +9,42 @@ further_reading:
 
-When the dashboard is no longer in "live" mode, the vertical pink line and the gray line indicating "Now" starts to diverge. In the chart below, the dashed pink line is still splitting the history from the forecast, but we can see actual data (solid green) to the right of the pink line. The lighter green is still showing the range of predicted values and the dotted green line is still showing the predicted future values. The forecast is based only on the actual data observed before the dashed pink line. This view is useful for checking how accurate the forecast is against known data.
+{{< img src="monitors/monitor_types/forecasts/query_editor.png" alt="query editor" responsive="true" style="width:80%;">}}
 
-{{< img src="monitors/monitor_types/forecasts/dashboard_past.png" alt="dashboard past" responsive="true" style="width:80%;">}}
+### Reading your dashboard
 
-To create a forecast graph, start by adding a timeseries graph to your dashboard. Be sure to select **Timeseries** as the visualization type. Now, click on the **+** icon on the right side of your expression. Choose the **Forecast** function in the **Algorithms** submenu. You should immediately see the preview update to include the forecast visualization. A number of the graphing options disappear, as forecasts have a unique visualization.
+The chart below shows a dashboard tile of a forecast in "live" mode. The dashed pink line divides the history from the forecast, and the current time is indicated by the gray "Now" marker and line. The dotted green line shows the future values predicted by the forecast, and the range of predicted values within three deviations (a tunable parameter) is shown in lighter green.
 
-The function has two parameters. The first parameter is for selecting which algorithm (see [Forecast Algorithms](#forecast-algorithms)) is used. The second parameter is deviations, and you can tune this to change the width of the range of forecasted values. A value of 1 or 2 should be large enough to accurately forecast most "normal" points. After successfully adding **Forecast**, your editor should show something like this:
+{{< img src="monitors/monitor_types/forecasts/dashboard_live.png" alt="dashboard live" responsive="true" style="width:80%;">}}
 
-{{< img src="monitors/monitor_types/forecasts/query_editor.png" alt="query editor" responsive="true" style="width:80%;">}}
+When the dashboard is no longer in "live" mode, the vertical pink line and the gray line indicating "Now" starts to diverge. In the chart below, the dashed pink line is still splitting the history from the forecast, but we can see actual data (solid green) to the right of the pink line. The lighter green is still showing the range of predicted values, and the dotted green line is still showing the predicted future values. The forecast is based only on the actual data observed before the dashed pink line. This view is useful for checking how accurate the forecast is against known data.
```

Comment:
```suggestion
When the dashboard is no longer in "live" mode, the vertical pink line and the gray line indicating "Now" starts to diverge. In the chart below, the dashed pink line is still splitting the history from the forecast, but the actual data (solid green) is to the right of the pink line. The lighter green is still showing the range of predicted values, and the dotted green line is still showing the predicted future values. The forecast is based only on the actual data observed before the dashed pink line. This view is useful for checking how accurate the forecast is against known data.
```

Reply:
Should "starts to diverge" be "start to diverge"?